### PR TITLE
Simpler solution to the problem using files.

### PR DESCRIPTION
**Pull request by Kush1101**

### CODE REVIEW

**Comment by nayuki** [comment]
**Refused** because your code has the following problems:

* Does not respect my formatting style for indentation, spacing, and naming.
* Contains timing logic, which my code does not have because it's in eulertest.py instead.
* Stripped out the main() runner that I architected.
* Uses inappropriate docstring syntax for comments.
* Did not supply the names.txt data file, hence this repository would not be self-contained.

Basically, you are attempting to shove your complete, independent, sloppy code into my project with total disregard to fitting into my body of existing work.

**Comment by Kush1101** [comment]
Hey Nayuki! Thank you for your comments. I'll be more careful with my PR's to anyone from next time.
I am a beginner and that's why my code couldn't match your high standards.

**Comment by nayuki** [comment]
Thanks for understanding.
